06-lowflow.Rmd

```diff
@@ -1,16 +1,779 @@
++-------------+--------------------------------------------------------+------------+
+
+The tables shows that the Frank copula with the reservoir index (RI) as the explanatory variable had the lowest AICc value of -31.10, indicating the best fit among the evaluated models. This is followed by the GH copula with an AICc of -30.16, and the Clayton copula with an AICc of -14.61. In comparison, when time was used as the explanatory variable, the Frank copula still performed best with an AICc of -29.41, but its fit was not as good as with the reservoir index.
+
+The results demonstrate that the Frank copula model, particularly when using the reservoir index as the explanatory variable, provides the most accurate representation of the time-varying dependence structure between the low-flow series at the two gauges. This highlights the significant impact of reservoir operations on the hydrological behavior of the Hanjiang River, emphasizing the importance of considering such anthropogenic factors in hydrological modeling.
+:::
+
+## Hybrid Dynamical–Statistical Approaches
+
+Hybrid dynamical–statistical (or statistical–dynamical) approaches offer several advantages over purely statistical or purely dynamical methods for predicting and projecting nonstationary extremes. In nonstationary hydrological analysis, a time or climate covariate can be used to detect changes in the distribution parameters. The benefit of using climate covariates is that climate model predictions or projections can then be utilized as covariates to estimate future changes.
+
+Hydrologists are interested in catchment-scale projections, but climate models operate at much larger resolutions, necessitating downscaling for use in process-based models. Due to the coarse spatial and temporal resolution of climate forecasts, downscaling is required to provide regional information at finer temporal and spatial scales [@Fowler2007, @Tian2017]. Various statistical downscaling and bias correction methods are employed. In the following section, we will present the Statistical Downscaling Model employed by Du et al. (2015) in the context of low-flow events.
+
+### Statistical Downscaling Model (SDSM)
+
+GCMs are tools used for predicting future time series of meteorological variables, thereby extending the time-varying exceedance probabilities described in Eqs. (13) and (15). The usefulness of GCMs for local impact studies is limited by their coarse spatial resolution, typically around 50,000 km², and their inability to resolve important sub-grid scale features such as clouds and topography which can be seen from the schematic illustration of general approach of SDSM Figure \@ref(fig:fig4) below. Therefore, the coarse spatial resolution of GCM data limits its direct application to local impact studies [@Wilby2002, @Wilby2007].
+
+```{r fig4, cache=FALSE, out.width="500", fig.align="center", echo=FALSE, eval = TRUE, fig.cap="Schematic illustration of SDSM"}
```

Comment:
add your name to the code chunk caption to avoid duplicated chunk labels
`fig4_gamze`

Reply:
Also for the follwoing chunks